lib/types/redux-types.js

```diff
@@ -792,6 +792,22 @@ export type BaseAction =
   | {|
       type: 'CREATE_LOCAL_MESSAGE',
       payload: LocallyComposedMessageInfo,
+    |}
+  | {|
+      type: 'SEND_FRIEND_REQUEST_STARTED',
+      payload?: void,
+      loadingInfo: LoadingInfo,
+    |}
+  | {|
+      type: 'SEND_FRIEND_REQUEST_SUCCESS',
+      payload: {},
```

Comment:
Just a heads-up, `{}` is actually pretty unsound in Flow (as is `Object`). Right now it looks like the payload is `void` (which is `typeof undefined`), but if you're trying to type an object, some options are `{||}` (enforced empty object) or `{| +[key: string]: mixed |}` (generic object with read-only properties)